src/state/profile/index.tsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
+import { Profile, ProfileState } from 'state/types'
+import getProfile from './getProfile'
+
+const initialState: ProfileState = {
+  isInitialized: false,
+  isLoading: true,
+  profile: null,
```

Comment:
I think the pattern we used on other states is `data`. Otherwise you'll have this structure `profile.profile`